benchopt/runner.py

```diff
@@ -314,7 +314,10 @@ def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
     Returns
     -------
     df : instance of pandas.DataFrame
-        The benchmark results.
+        The benchmark results. If multiple metrics were computed, each
+        one is stored in a separate column. If the number of metrics computed
+        by the objective is not the same for all parameters, the missing data
+        is set to `NaN`.
```

Comment:
Not sure if we want to add this. The computation should not output NaN in principle as the metric is always computed the same no?

Reply:
The computation does not make NaNs. The conversion to csv by pandas does.
It's like in the test situation. It is possible to compare 1 metric for parameter a and 2 metrics for parameter b.
In that case, the final csv file contains the columns of the dic (one for each metric). But the first parameter doesn't have any value to put in the second metric column. So this part of the column is filled with NaNs by pandas.

Reply:
Ok